<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remove_book</class>
 <widget class="QDialog" name="remove_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/images/bg_login2.jpg&quot;);
background-position: center</string>
  </property>
  <widget class="QPushButton" name="btn_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 82, 223);
font: 10pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLabel" name="texto_frase">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,79,0);
font: 9pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>&quot;Um livro é um sonho que você segura com
              as mãos.&quot;  - Neil Gaiman</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>141</y>
     <width>376</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="txt_inicio">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 14pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Insira título e autor do livro que
você deseja remover.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="txt_titulo">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Título</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="insert_titulo"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="txt_autor">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Autor</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="insert_autor"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="txt_formato">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Formato</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="insert_formato">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhNone</set>
      </property>
      <item>
       <property name="text">
        <string>Fisico</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Digital</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audio</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="txt_aviso">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Atenção: a exclusão do livro é permanente. Você precisará adicioná-lo de novo,
se preciso.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="btn_remover">
      <property name="styleSheet">
       <string notr="true">background: rgb(80, 0, 0);
color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="txt_resultado">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
